src/commands/workspace/start.ts

```diff
@@ -8,72 +8,29 @@
  * SPDX-License-Identifier: EPL-2.0
  **********************************************************************/
 
-import { Command, flags } from '@oclif/command'
-import { string } from '@oclif/parser/lib/flags'
 import { cli } from 'cli-ux'
+import * as notifier from 'node-notifier'
 
-import { CheHelper } from '../../api/che'
-import { accessToken, cheNamespace, listrRenderer } from '../../common-flags'
-export default class Start extends Command {
-  static description = 'create and start a workspace'
+import Create from './create'
 
-  static flags = {
-    help: flags.help({ char: 'h' }),
-    chenamespace: cheNamespace,
-    devfile: string({
-      char: 'f',
-      description: 'path or URL to a valid devfile',
-      env: 'DEVFILE_PATH',
-      required: true,
-    }),
-    name: string({
-      description: 'workspace name: overrides the workspace name to use instead of the one defined in the devfile. Works only for devfile',
-      required: false,
-    }),
-    'access-token': accessToken,
-    'listr-renderer': listrRenderer
-  }
-
-  async checkToken(flags: any, ctx: any) {
-    if (ctx.isAuthEnabled && !flags['access-token']) {
-      this.error('E_AUTH_REQUIRED - Eclipse Che authentication is enabled and an access token need to be provided (flag --access-token).')
-    }
-  }
+export default class Start extends Create {
+  static description = 'Creates and starts workspace from devfile'
```

Comment:
I believe that this command is still used.
So, let's keep the current functionality + add warning...

Reply:
That's exactly what I do.